pkg/constants/constants.go

```diff
@@ -60,4 +60,7 @@ const (
 
 	VersionPath         = "/bpf_kmesh/map/"
 	WorkloadVersionPath = "/bpf_kmesh_workload/map/"
+
+	Tail_call_map = "tail_call_map"
```

Comment:
```suggestion
	TailCallMap = "tail_call_map"
```
In go, everyone uses camel naming by default.